.travis.yml

```diff
@@ -8,6 +8,7 @@ matrix:
     - {python: 3.4, env: TOX_ENV=py34}
     - {python: 3.5, env: TOX_ENV=py35}
     - {python: 3.6, env: TOX_ENV=py36}
+    - {python: 3.7, env: TOX_ENV=py37, dist: xenial, sudo: true}
```

Comment:
I'm guessing we need xenial because earlier versions don't have python 3.7. Why the sudo?

Reply:
I took it from https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-401756442

Reply:
We can try it without `sudo` and see if it works.